src/java/org/apache/cassandra/cql3/UpdateParameters.java

```diff
@@ -274,6 +259,14 @@ public Row buildRow()
         return built;
     }
 
+    @Override
+    public void close()
+    {
+        if (builder == null) return;
+        builder.reset(); // Clear resources
```

Comment:
this logic isn't needed as much anymore, but if the row was rejected we are able to clear it up and put back into the pool and reuse the memory.